tests/formatter/test_project.py

```diff
@@ -101,9 +101,7 @@ def test_entry_points(fmt: Fmt) -> None:
     """
     expected = """
     [project.entry-points]
-    alpha = {"A.A" = "a",B = "b"
-    }
-    beta = {C = "c",D = "d"
-    }
+    alpha = {"A.A" = "a",B = "b"}
```

Comment:
This reads badly, can we add a space between `,` for inline tables? 🤔 while we're at it.

Reply:
I agree, I was about to ask if that was an stylistic decision, but I suppose it was not 😄 

I can give it a try, but currently `tomlkit` does not support that. So any solution I propose might sound hacky (I am afraid).

Reply:
@gaborbernat, I gave it a try in https://github.com/abravalheri/pyproject-fmt/pull/2, but the outcome was not very good.

Indeed, when I solve this problem, I introduce a different one.

I think the best course of action here is to add the sorting capability directly in `tomlkit`, then proceed doing the change here.

Reply:
We'll have to follow-up on that then, let's accept this as is, one bugfix at a time.